ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/constants/page-url.constants.ts

```diff
@@ -1,6 +1,6 @@
 export let PageUrls = {
     PageNotFound: '/page-not-found',
-    Unauthorised: '/unauthorised',
+    Unauthorised: '/unauthorized',
```

Comment:
Should the key be changed as well? If anything I would have thought the key should be changed but the url left the same as this will be visible to users.

Reply:
good eye, will check but it seems we have multiple files for page URLs which is a bit of a pain